src/main/webapp/js/TextEditor.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Initialize the iframe into a RTC
+ * TODO: CSS the frame
+ */
+function init(id)
+{
+	//check for null param
+	id = id || "";
+	//Gets the document of iframe
+	var doc = document.getElementById(id).contentDocument; 
+
+	//Make iframe into a text editor
+	doc.designMode = "on";
+	doc.contentEditable = true;
+	doc.write('<html><head><style>body{ margin:3px; word-wrap:break-word; word-break: break-all; }</style></head><body>CodeUtest</body></html>') ;
+};
+//TODO: make into a switch statement while adding more functions
+function setBold(id)
+{
+	var currWin = document.getElementById(id).contentWindow;
+	currWin.document.execCommand("Bold", false, null);
```

Comment:
Is there a way to return focus back to the typing window once this is run?

Reply:
Hmm, I think it will be better to let users know where exactly did they do the formatting, is that what you mean or did I misunderstood?